model/data_test.go

```diff
@@ -22,3 +24,33 @@ func TestAddReflogHash(t *testing.T) {
 func (g GitCommanderMock) Reflog(opt string) ([]byte, error) {
 	return []byte("a517e3b\na517e3b\n0b860a1\na517e3b\na517e3b\n0b860a1"), nil
 }
+
+func TestAddcommitMessage(t *testing.T) {
+	gci := SetGitExecuter(GitCommanderMock{}) // use mock for unit test
+	gci.AddCommitMessage("SpecifiedMsg")
+
+	// g.CommitMsgList = append(g.CommitMsgList, v)
+	got := len(gci.CommitMsgList)
+	want := 6
+	if got != want {
+		t.Fatalf("want %v, but %v:", want, got)
+	}
+
+	// g.CommitNewMsgList = append(g.CommitNewMsgList, fmt.Sprintf("fixup! %s", v))
+	gots := strings.Join(gci.CommitNewMsgList, " ")
```

Comment:
Why do you use `gots` value?
`rwant` is also.

Reply:
Please add comments details to explain the what this code tests.

Reply:
addressed and force pushed